src/TodoForm.jsx

```diff
@@ -1,22 +1,35 @@
+import { useState } from 'react';
 import { useRef } from 'react';
 
 function TodoForm({ onAddTodo }) {
-  const todoTitleInput = useRef(null);
+  const [workingTodoTitle, setWorkingTodoTitle] = useState('');
+
+  const inputRef = useRef(null);
 
   function handleAddTodo(event) {
     event.preventDefault();
-    const title = event.target.title.value;
+    const title = workingTodoTitle.trim();
+    if (title === '') return;
     onAddTodo(title);
+    setWorkingTodoTitle('');
 
-    event.target.title.value = '';
-    todoTitleInput.current.focus();
+    if (inputRef.current) inputRef.current.focus();
```

Comment:
This guard is not necessary. Since you will always call `handleAddTodo` from  the same `input` element. `inputRef.current` will always be a truthy object when your handler function is called.